theano/tensor/elemwise.py

```diff
@@ -502,6 +503,7 @@ class Elemwise(OpenMPOp):
     | Elemwise(log)(rand(3, 4, 5))
 
     """
+    __props__ = ("scalar_op", "inplace_pattern", "name", "nfunc_spec", "openmp")
```

Comment:
Yes, I ignore this class for caching. So, of the places that i've added `__props__` which ones should i retain and which one should i remove ? I did not make any change to the old backend. There are a couple of Op at least that do not have `__props__` Do i add there as well ?